rllib/core/learner/learner.py

```diff
@@ -1014,6 +1009,12 @@ def _check_if_build_called(self):
                 "Learner and before calling any methods on it."
             )
 
+    def _reset(self):
```

Comment:
jooc: Why did this get its own method?

Reply:
for reducing cognitive load -- I'm essentially trying to tell devs that I'm resetting these states.

Reply:
Got it. I saw that K requested it but didn't understand why. Thanks!